dev-client/src/components/ListFilter.tsx

```diff
@@ -296,12 +296,14 @@ type Props = Omit<
   'value' | 'onValueChange' | 'nullable'
 > & {
   name: string;
+  nullable?: boolean;
 };
 
 export const SelectFilter = ({
   name,
   options,
   label,
+  nullable = true,
```

Comment:
An optional boolean parameter that defaults to true seems counterintuitive to me. Is there a reason to not make it default to false?

Reply:
I don't have a take on this myself, but I'm guessing that it's this way because prior to this change, it was effectively always true for `SelectFilter`s

Reply:
I made it true to preserve the previous behaviour. Want me to make it false and update existing files?

Reply:
I did that.